docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -82,16 +82,17 @@ successfully built for each component that is to be updated in this release.
 
     make all
 
-## 7. Produce distribution tarballs
+For all components, make sure that all files are covered by MANIFEST and/or 
+MANIFEST.SKIP:
 
-In this step produce the *preliminary distribution tarballs*.
+    make distcheck
 
-For all components, make sure that all files are covered by MANIFEST and/or MANIFEST.SKIP:
+> For all components, review the MANIFEST file if in doubt or if there seems to be
+> missing or extra files.
```

Comment:
Do we really need an explicit note about this?

Reply:
@mattias-p! Not really. I just kept the text from previous version. I'll update.